Server/Utilities/SpringBoardAlerts.m

```diff
@@ -45,28 +41,99 @@ - (instancetype)init {
                                  userInfo:nil];
 }
 
++ (void)raiseIfInvalidAlert:(NSDictionary *)alertDict
+                 ofLanguage:(NSString*)language
+                andPosition:(NSInteger)position {
+    if (!alertDict[@"title"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No title"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (!alertDict[@"buttons"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No buttons"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (((NSArray *)alertDict[@"buttons"]).count == 0) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"Zero size buttons array"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (!alertDict[@"shouldAccept"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No shouldAccept"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+}
+
++ (NSArray<NSString*> *) getLanguagesFromBundle:(NSBundle*) bundle {
```

Comment:
Avoid using `get` in Objective-C except when the method returns a value indirectly.

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html

```
- (void)getLineDash:(float *)pattern count:(int *)count phase:(float *)phase;
```

Reply:
ok